delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -264,7 +295,7 @@ def checkoutSCM(bitbucket_repo_name, var_cas_clone_url) {
  * @return
  */
 def loadServerlessConfig(runtime) {
-	//events.sendStartedEvent('GET_SERVERLESS_CONF')
+	events.sendStartedEvent('GET_SERVERLESS_CONF')
```

Comment:
check if you can convert all these event names to enumerations. Harcoded strings is not a good practice. If we have to make a change, we have to do this everywhere instead of one location in the code.

Reply:
I will change the hardcoded strings and make use of enumerations.